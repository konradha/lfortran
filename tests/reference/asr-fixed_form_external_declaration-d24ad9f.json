{
    "basename": "asr-fixed_form_external_declaration-d24ad9f",
    "cmd": "lfortran --indent --fixed-form --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_external_declaration.f",
    "infile_hash": "17a8e11b1aeaaa6f6e5e286ef8a5afc45fa279c7bc7a5a7a63d0abc2",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-fixed_form_external_declaration-d24ad9f.stdout",
    "stdout_hash": "379f6c6b04840ff9a42736942306d8ed48ec5b8792f0a7d6235d0bed",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}