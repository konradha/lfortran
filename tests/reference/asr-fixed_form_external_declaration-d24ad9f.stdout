(TranslationUnit 
    (SymbolTable
        1
        {
            dm:
                (Function 
                    (SymbolTable
                        3
                        {
                            
                        }) 
                    dm 
                    [(StringConstant 
                        "e" 
                        (Character 1 1 () [])
                    )] 
                    [(Character 1 1 () [])] 
                    [] 
                    () 
                    Source 
                    Public 
                    Interface 
                    () 
                    .false. 
                    .false. 
                    .false. 
                    .false.
                ), 
            main:
                (Program 
                    (SymbolTable
                        2
                        {
                            dm:
                                (Variable 
                                    2 
                                    dm 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Real 8 []) 
                                    Source 
                                    Public 
                                    Required 
                                    .false.
                                )
                            
                        }) 
                    main 
                    [] 
                    [(Print 
                        () 
                        [(FunctionCall 
                            1 dm 
                            () 
                            [((StringConstant 
                                "e" 
                                (Character 1 1 () [])
                            ))] 
                            (Character 1 1 () []) 
                            () 
                            ()
                        )] 
                        () 
                        ()
                    )]
                )
            
        }) 
    []
)
